### DISCUSSION TITLE
Fixed cable, start charging on connected

### DISCUSSION THREAD

**Discussion by HA-TB303** [dingo35/SmartEVSE-3.5#25] [Q&A] [answered]
Hi, 

We can't seem to be able to figure out how to start the charging proces when connecting the car using a fixed cable.
The SmartEVSE 3 is running version v3.5.0-RC. Nothing connected except a 4p contactor and a 3 phase modbus meter. No RFID, no switch. We just want it to start charging when a car is connected.
We are feeding the phase net-usage information into the SmartEVSE through the API.

Screenshot:
![CleanShot 2024-02-29 at 21 58 29](https://github.com/dingo35/SmartEVSE-3.5/assets/9609805/cfb3c37a-a20e-46b6-b003-bfb62788d3a8)

There is no car connected at this moment and the actual net-usage on the 3 phases is indeed:
L1: 9.0 A
L2: 8.0 A
L3: 7.0 A

When I connect the car (in this case a Tesla model 3 and a Renault Megane Hybrid), the SmartEVSE detects the car, but does not start charging. Also not when I activate it through Home Assistant like:

![CleanShot 2024-02-29 at 22 01 43](https://github.com/dingo35/SmartEVSE-3.5/assets/9609805/3fd01ed7-df52-43c1-a564-613f001ff75e)

We have only 80amps per phase available for the building and the 4 SmartEVSE chargers, so the chargers can only do 10amp per phase, so that is why it is set to only 10A and the charges are secured with a 10A circuit breaker.

The only way to get the charging to start is to press 'reboot'  from the ui.

You guidance is much appreciated!

**Comment by dingo35** [comment]
Does Normal charging with the original v3.0.1 firmware from the manufacturer work?

**Reply by HA-TB303** [reply]
Hi, just tested with the original firmware and yes, it starts charging right away. Sadly no modbus EV meter support and no smart charging then.

I might have a lead on the root cause. Could it be that I need to feed the remaining Amps available on each phase from my P1 meter? I now just feed the used amps per fase with esphome:

```
interval:
  - interval: 4sec
    then:
      - http_request.post: 
          url: !lambda |-
              std::string url;
              url.append("http://192.168.228.111/currents?L1=");
              url.append(to_string(id(currentl1).state *10));
              url.append("&L2=");
              url.append(to_string(id(currentl2).state *10));
              url.append("&L3=");
              url.append(to_string(id(currentl3).state *10));
              return url.c_str();

```

I can emagine the SmartEVSE want's to know what is actually left on the phase for charging, in this case for example 
L1 = 80Amps minus current1

There is no clear documentation about that.

**Reply by HA-TB303** [reply, marked as answer]
grumpf....fixed ;)

I had:
```
MAINS	(only appears when a MAINSMET is configured):
        Set Max Mains current: 10-200A (per phase)
```

set to 10A because I thought this had to be the limit of the charger
